### DISCUSSION TITLE
Possibility to see first lines of the article content underneath the title

### DISCUSSION THREAD

**Discussion by cnstudios** [FreshRSS/FreshRSS#4752] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Nowadays the titles of articles are often used for clickbait, so it would be nice to see underneath the titles in the main view a few lines of the beginning of the article as to decide to wether it really is worth reading. 

**Describe the solution you’d like**
I'd like to see maybe the first three lines (or maybe the first x characters) of the article content underneath the article title in the main view.

**Describe alternatives you’ve considered**
For now I'm opening more articles to read the first segment and decide wether it really is as interesting as the title makes it to be.

**Additional context**

**Comment by Alkarex** [comment, marked as answer]
Please try the *Summary* option:

![image](https://user-images.githubusercontent.com/1008324/196771160-3677a9e2-0508-4c35-974b-80a4569c382a.png)


It also works well with the *Thumbnail* option:

![image](https://user-images.githubusercontent.com/1008324/196771569-c3c0e08e-d01d-4d50-b308-7bb54dad18fd.png)

**Reply by cnstudios** [reply]
Great! I missed that option! Exactly what I was looking for! Thanks so much! :-)
